packages/test/test-utils/src/loaderContainerTracker.ts

```diff
@@ -32,6 +33,7 @@ interface ContainerRecord {
 export class LoaderContainerTracker implements IOpProcessingController {
     private readonly containers = new Map<IContainer, ContainerRecord>();
     private lastProposalSeqNum: number = 0;
+    private readonly defaultTimeoutDurationMs: number = 2000;
```

Comment:
@anthony-murphy Considering we can not break the API signature to add a new param for timeout, I set the default timeout duration as a private member variable, and use 2s as default value (which is the same as default Mocha test timeout)

Reply:
Could you just add a new method that takes a timeout? Calling the old one can not limit time/use large value.